docs/source/NEWS.rst

```diff
@@ -1,6 +1,19 @@
 Changelog
 =========
 
+Release 25.0.0 (unspecified)
+----------------------------
+
+Features
+^^^^^^^^
+
+- Client sessions support
```

Comment:
Should we put something here to indicate initial or partial support?

Reply:
Added more detailed description, including limitations